transforms/universal/noop/kfp_ray/Makefile

```diff
@@ -49,3 +49,7 @@ workflow-upload: workflow-build
 	@for file in $(YAML_WF); do \
 		$(MAKE) .workflows.upload-pipeline PIPELINE_FILE=$$file; \
 	done
+
+.PHONY: workflow-generate
+workflow-generate:
+	../../../../kfp/pipeline_generator/single-pipeline/run.sh -c `pwd`/pipeline_definitions.yaml -od  .
```

Comment:
I had to add `pip install pyyaml` to run kfp/pipeline_generator/single-pipeline/run.sh